pre-deploy.js

```diff
@@ -49,26 +49,27 @@ Promise.all([
 
 function download (file) {
   return new Promise((resolve, reject) => {
-    request({
-      url: `http://unpkg.com/vue@${version}/dist/${file}`,
-      encoding: null
-    }, (err, res, body) => {
-      if (err) {
-        return reject(err)
-      }
-      if (res.statusCode != 200) {
-        return reject(
-          `unexpected response code when downloading from unpkg: ${res.statusCode}` +
-          `\n${body.toString()}`
-        )
-      }
-      fs.writeFile(`themes/vue/source/js/${file}`, body, err => {
-        if (err) return reject(err)
-        zlib.gzip(body, (err, zipped) => {
+    fetch(`http://unpkg.com/vue@${version}/dist/${file}`)
```

Comment:
```suggestion
    fetch(`https://unpkg.com/vue@${version}/dist/${file}`)
```

Might as well use HTTPS now.